docs/source/contributing/developing.adoc

```diff
@@ -258,6 +258,14 @@ $ make integration GODOG_OPTS="-paths ~/tests/custom.feature,~/my.feature -tags
 NOTE: When multiple values are used for options in `GODOG_OPTS`, then they have to be separated by a comma without whitespace.
 While `-tags basic,openshift` will be parsed properly by make, `-tags basic, openshift` will result in only `@basic` being used.
 
+==== Viewing Results
+
+Integration test logs its progress directly into a console with accent of providing additional useful information when failures of individual Gherkin steps happen.
```

Comment:
"Integration tests log per default to the console, providing basic test progress information. This information is often enough to find and debug failure reasons. However, for cases needing further investigation, the integration tests also log more detailed information to  the log file `$GOPATH/github.com/minishift/minishift/out/integration-test/integration.log`."

Maybe something like that.